docs/docsite/rst/dev_guide/developing_collections_structure.rst

```diff
@@ -175,7 +175,11 @@ You can migrate 'traditional roles' into a collection but they must follow the r
 playbooks directory
 --------------------
 
-TBD.
+Since 2.9 you can reference playbooks in this directory, but it requires the full path to the file from the command line.
+Since ansible-core 2.11 you can use the FQCN, ``namespace.collection.playbook`` (with or without extension), to reference them from the command line or from ``import_playbook``.
+This will keep the playbook in 'collection context', as if you had added ``collections: [ namespace.collection ]`` to it.
+
+You can have most of the subdirectories you would expect, like ``files/``, ``vars/`` or  ``templates/`` but no ``roles/`` since those are handled already in the collection.
```

Comment:
```suggestion
You can have most of the subdirectories you would expect, such ``files/``, ``vars/`` or  ``templates/`` but no ``roles/`` since those are handled already in the collection.
```